.github/workflows/release-pr.yml

```diff
@@ -11,6 +11,14 @@ on:
         description: "Helm version to release"
         required: true
         default: "0.0.0"
+      k8s_versions:
+        description: "Support Kubernetes versions for this release"
```

Comment:
I think this could be changed to something like:
`description: "kubernetes versions this release has been tested on"`

Reply:
Thanks @vepatel I have made those changes